backend/saas-migrations/frontline/commands/migrateChannel.ts

```
@@ -6,19 +6,49 @@ const MONGO_URL =
+if (!TARGET_SUBDOMAIN) {
+  throw new Error('Environment variable TARGET_SUBDOMAIN must be set.');
+}
+
+function extractDbName(url) {
+  const withoutQuery = url.split('?')[0];
+  return withoutQuery.slice(withoutQuery.lastIndexOf('/') + 1);
+}
+
+const client = new MongoClient(CORE_MONGO_URL || MONGO_URL);
 
 let db;
```

Biome (2.5.0)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)